package.json

```diff
@@ -143,6 +143,7 @@
         "@types/redux-devtools-extension": "2.13.2",
         "@types/testing-library__cypress": "5.0.6",
         "cypress": "5.0.0",
+        "cypress-commands": "^1.1.0",
```

Comment:
As far as I can see your not defining any custom commands so this seems to be unnecessary (at least for the moment)

Reply:
Sorry, that's wrong. I'm using the `.attribute()` selector in the cypress test to test the title attribute of the status-bar length. That selector isn't included in the core-package of cypress but is in the cypress-commands package. See [the docs](https://github.com/Lakitna/cypress-commands/blob/develop/docs/attribute.md#attribute) for details about it.